### PR TITLE
Fix grammatical issues in node.md

### PR DESCRIPTION
**Pull request by ghodss**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1570)

### CODE REVIEW

**Comment by idvoretskyi** [comment]
LGTM

File: docs/admin/node.md [hunk 1]
```diff
@@ -11,44 +11,39 @@ assignees:
...
 architecture design doc for more details.
 
 ## Node Status
 
-Node status describes current status of a node. For now, there are the following
-pieces of information:
+A node's status is comprised of the following information.
 
-### Node Addresses
+### Addresses
```
Comment (techtonik) [inline docs/admin/node.md]: Not sure if it is better to leave it as-is for better indexing.

**Comment by lavalamp** [comment]
Sorry, I was OOO and it looks like this needs a rebase.

**Comment by devin-donnelly** [comment]
Please resolve the merge conflict.

**Comment by jaredbhatti** [comment]
Please Rebase so we can merge.

**Comment by jaredbhatti** [comment]
Last request for rebase.

**Comment by ghodss** [comment]
@jaredbhatti This is my 2nd rebase, please let me know if you need anything else to merge.

**Comment by jaredbhatti** [comment]
Thanks!

**Pull request merged**